test/compress/harmony.js

```diff
@@ -583,3 +583,39 @@ class_extends_regex: {
     }
     expect_exact: "function f(){class rx1 extends(/rx/){}}"
 }
+
+issue_2028: {
+    options = {
+        side_effects: true,
```

Comment:
Does this test still work with `unused=true` and the other default compress options?